hw/ip/otp_ctrl/doc/dv/index.md

```diff
@@ -98,16 +98,9 @@ Some of the most commonly used tasks / functions are as follows:
 
 #### Functional coverage
 To ensure high quality constrained random stimulus, it is necessary to develop a functional coverage model.
-The following covergroups have been developed to prove that the test intent has been adequately met:
-* power_on_cg: This covergroup collects lc_escalation_en signal and partitions lock status after OTP_CTRL initialization is done.
-  Note that this covergroup only samples data after scoreboard detected OTP_CTRL's initialization is successful and checks are passed.
-* flash_req_cg: This covergroup collects two different flash requests(addr and data) and secret1 partition lock status.
-  Note that this covergroup only samples data after scoreboard detected that flash request is finished and checks are passed.
-* sram_req_cg: This covergroup collects two different sram requests and secret1 partition lock status.
-  Note that this covergroup only samples data after scoreboard detected that sram request is finished and checks are passed.
-* otbn_req_cg: This covergroup collects secret1 partition lock status when otbn request is finished.
-  Note that this covergroup only samples data after scoreboard detected that otbn request is finished and checks are passed.
-* lc_prog_cg: This covergroup collects if lc programming request finished with or without error.
+The following two files declared OTP_CTRL's covergroups:
+- `dv/env/otp_ctrl_env_cov.sv`: This file declares functional or CSR related covergroups. The functional coverage is collected manually inside OTP_CTRL's scoreboard by invoking the `sample` function.
```

Comment:
```suggestion
- `dv/env/otp_ctrl_env_cov.sv` declares functional or CSR related covergroups.
  The functional coverage is collected manually inside OTP_CTRL's scoreboard by invoking the `sample` function.
```

Reply:
That sounds much more condense, thanks so much